locales/en/common.json

```diff
@@ -31,5 +31,6 @@
   "type_value_to_confirm_plain": "Type {{value}} to confirm",
   "unexpected_error": "There was a problem processing the request. Please try again.",
   "updated": "Updated",
-  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for."
+  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for.",
+  "select": "Select"
```

Comment:
Please run `npm run parse-i18n` when you add keys, so to have them sorted alphabetically and avoid conflicts in future PRs if someone runs that command.